### ISSUE TITLE
Repository Abandoned?

### ISSUE THREAD

**Issue by tomaswitek** [malte-wessel/react-custom-scrollbars#275]
Hi @malte-wessel first of all thx for your wonderful work.
I would like to fix couple of issues since I am using it in production.
I see a lot of PRs and no progress on them.
Do you plan to merge those PRs or are you searching for a new maintainer?

**Comment by llamamoray** [comment]
@tomaswitek I'm in the same position. It's been a long while since anything has changed on this repo - do you fancy forking and creating a new product? Looking at the issue tracker and PRs there are plenty of us willing to chip in with maintenance.

**Comment by Rafi993** [comment]
It would be awesome if there is any update on this

**Comment by bristoljon** [comment]
Yeah this is still the best react scroll bars solution I've come across despite the bugs. Maybe could just pick a fork and notify pull request author's to apply them to forked repo then publish under new name?

**Comment by malte-wessel** [comment]
Hey guys, sorry responding so late, I will try to work through all the issues and PRs in the next couple of days/weeks!

**Comment by tomaswitek** [comment]
Thank you 🙌

**Comment by terry90** [comment]
Nice ! :+1:

**Comment by avkonst** [comment]
> Hey guys, sorry responding so late, I will try to work through all the issues and PRs in the next couple of days/weeks!

Thanks. Did you have a chance to consider the PRs?

**Comment by G-Rath** [comment]
@malte-wessel I'd be more than happy to help out in my free time - ping me if you're interested, and we can figure out the best way to tackle things 😄 

I can start by looking through the existing PRs, and attempt to review them (which I think would be the best starting point to get things to back on track, followed by some issue triaging).

**Comment by camdagr8** [comment]
Has anyone decided to make an official fork of this yet?

**Comment by sattaman** [comment]
@malte-wessel Could you post asking for maintainers?

**Comment by RickeyWard** [comment]
@malte-wessel We would love to see some motion on this repo. I know it's hard to maintain an open source project after you've used it for your purposes but we really appreciate this one and it's forming a community. We are willing to donate some of our time, if it would help.

**Comment by Tomassito** [comment]
> Has anyone decided to make an official fork of this yet?

Guess it's high time to do it!
https://github.com/Tomassito/react-custom-scrollbars
Any easy way to copy/move PRs or need to ask each PR contributor to redo them on the fork?
Who's up to be co-maintainer? 
@tomaswitek 
@G-Rath ?

**Comment by G-Rath** [comment]
@Tomassito Always happy to be added as a maintainer to things 😄

**Comment by Tomassito** [comment]
Cool! So how are we proceeding with reviving this project? In the meantime I found this: 
https://github.com/rejuvenate/rejuvenate/blob/master/README.md

**Comment by TPS** [comment]
Who here would actively maintain a fork?  I'd be happy to open 1 up per https://github.com/rejuvenate/rejuvenate/issues/6.

**Comment by G-Rath** [comment]
@TPS I would be happy to be involved in maintaining a fork.

My bandwidth wanes & waxes, so my code contributions might not be high, but I'm more than happy to help out w/ issue triaging, sounding board, PR review etc 🙂

**Comment by TPS** [comment]
I forgot to mention:  if anyone _already_ maintains a more-active fork (but doesn't want sole-maintainer responsibilities), I'd consider candidates to fork _from_.

**Comment by RickeyWard** [comment]
I'd prefer @malte-wessel just allow some maintainers. It's hard to call any fork official.

**Comment by Tomassito** [comment]
> I'd prefer @malte-wessel just allow some maintainers. It's hard to call any fork official.

Like everyone else here, but what are the chances, you reckon?

**Comment by RickeyWard** [comment]
I sent him a tweet, maybe he will respond there.

**Comment by TPS** [comment]
@RickeyWard Did you get any response?

Anyone else who wants in as devs/maintainers?  Or wants to volunteer a specific base to fork?

**Comment by RickeyWard** [comment]
![411D79CB-7FC7-4015-A177-FD8A6F5F310F](https://user-images.githubusercontent.com/342468/70381504-6655d780-1919-11ea-9701-6c4b07946b95.jpeg)


He replied, hopefully we see some movement

**Comment by malte-wessel** [comment]
Hey guys, sorry for the trouble. I'm open for contributors. Would be great if some experienced developer could take over? Any suggestions?

**Comment by RickeyWard** [comment]
I'm happy to help with the maintenance.

**Comment by Tomassito** [comment]
I could dedicate some of my time too. Using the custom scrolls in production, so I'm actively interested in the project's well-being.

**Comment by diragb** [comment]
any movement?

**Comment by sakhnyuk** [comment]
@malte-wessel , could you add me to a collaborator of this project? 
I have a copy of your code and using it in production. I moved component to Typescript and add some fixes. 
I can provide all my fixes to this project and keep it up to date

**Comment by brandonwisnicki** [comment]
Any updates? It's a real shame if this bites the dust. Really is one of the better scroll libraries out there.

**Comment by TPS** [comment]
I was relooking into opening up a `rejuvenate` fork (as above), but I didn't see enough _dev_ interest to proceed, & @malte-wessel was wanting collaborators, also.  Anyone?

**Comment by llamamoray** [comment]
@TPS sorry I missed your call for help a while ago.

I'm happy to give some time to the project. @sakhnyuk @G-Rath @tomaswitek are you still interested?

**Comment by sakhnyuk** [comment]
@llamamoray sure! I have updated code of that project. So I can try to fork and merge my updates.

**Comment by TPS** [comment]
@malte-wessel Are you still planning to add collaborators or perhaps transfer ownership?

@ Everyone else:  Failing that, should we fork from _here_, or are there better forks that y'all are using?  If such, is there a reason to add _another_ fork, or would everyone just prefer to move to those?

**Comment by Tomassito** [comment]
I'm happy to put in some work with reviews and improvements too. Can't really see that through my profile but I work with JS/TS almost every day.

**Comment by brandonwisnicki** [comment]
@TPS If there are more updated forks, I think it is worthwhile to not refix things starting from original repo. So long as the newer forks are have drop-in support, I see no harm.

I can help out a bit too, if needed. There's definitely a strong future to this library given the support.

**Comment by TPS** [comment]
> If there are more updated forks, I think it is worthwhile to not refix things starting from original repo. So long as the newer forks are have drop-in support, I see no harm.

@brandonwisnicki I agree.

Is there a consensus on _which_ fork?  Per above, @Tomassito & @sakhnyuk are each maintaining current production code of this, & perhaps others.

**Comment by Tomassito** [comment]
We don't currently have any other fork candidates than the one of @sakhnyuk, do we? 
Mikhail, do you mind sharing that with us so we can see how much you have changed? Do you know if it is backward compatible? 
@malte-wessel - man still waiting for your input here!

**Comment by sakhnyuk** [comment]
@Tomassito no problem! Let me few days for prepare project. 
I didn’t change API, so I think it would be fully backward compatible

**Comment by TPS** [comment]
@Tomassito Do you indeed _not_ have a current production fork _(anymore?)_, IIUC?  Is @sakhnyuk's the only current 1 now?

If so, @sakhnyuk, is there any reason _(once you've prepped the repo, as mentioned)_ for everyone to _not_ just migrate to yours, & you can assign your own collaborators, &c?  Is there actually a need for an additional `rejuvenate` fork?

**Comment by Tomassito** [comment]
Correct, I use it in production but not as a fork but in a wrapper to adjust/extend the api. 
In the meantime I wrote to Malte on Twitter couple of days ago again but no response so far :/.

**Comment by pitops** [comment]
Any updates guys?

**Comment by sakhnyuk** [comment]
@pitops Yep! I started working on project. I think on weekends I'll send link here.

**Comment by sakhnyuk** [comment]
Hey guys!
I created repo with my updated code.
Here you go:
https://github.com/sakhnyuk/rc-scrollbars
https://www.npmjs.com/package/rc-scrollbars

Project name on NPM is **rc-scrollbars**

What's done:
- Update all dependencies
- Move component on TS (with types definitions)
- Replace Babel on TSC
- Rewrite example project (web-page)

What need to work on:
- Tests (need to update Karma or move to cypress)
- example [page](https://rc-scrollbars.vercel.app/) (need move files to TS)

Feel free to open Issues and Pull requests.

**Comment by TPS** [comment]
@Tomassito Given the above, is it sufficient to close https://github.com/rejuvenate/rejuvenate/issues/6?
